apps/crn-server/src/entities/research-output.ts

```diff
@@ -136,6 +136,13 @@ export const parseGraphQLResearchOutput = (
         title: group.flatData.title || '',
       })) || [],
     published: output.status === EnrichedContentEventType.Published,
+    reviewRequestedBy: data.reviewRequestedBy?.[0]
```

Comment:
I'm not very fond of the idea of having `reviewRequestedBy` as a string when we update the research output and as an object when we get the research output data. But since it would give some work to change it, I would like to see what other people think about this @peterstarling @lennym @Riccardo-Brown